### PR TITLE
Template update v2.2

### PR DESCRIPTION
**Pull request by ggabernet**
- Template update to nf-core/tools v2.2
- updated module structure to match new module template

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/epitopeprediction/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/epitopeprediction _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -48,7 +55,13 @@ jobs:
       NXF_ANSI_LOG: false
     strategy:
       matrix:
-        nxf_ver: ["21.10.4", ""]
+        include:
+          # Test pipeline minimum Nextflow version
+          - NXF_VER: '21.10.3'
```
Comment (marissaDubbelaar) [inline .github/workflows/ci.yml:60]: Why nextflow version 21.10.3 instead of 21.10.4?

**Review by marissaDubbelaar — APPROVED** [review]

**Comment by marissaDubbelaar** [comment]
But it looks good to me!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ggabernet) [reply .github/workflows/ci.yml:60]: Ah right I mean this is the minimal version that should be tested, then the latest is also tested. So the minimal NXF_VER that still works for the pipeline should be cited here

**Review by christopher-mohr — APPROVED** [review]
One minor question but looks good to me 👍.

File: docs/images/nf-core-epitopeprediction_logo_dark.png [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
```
[the remark this answers is not in the document]
Comment (ggabernet) [reply docs/images/nf-core-epitopeprediction_logo_dark.png:1]: yeah right, or maybe adding the light logo here? At least sth will be displayed

[the remark this answers is not in the document]
Comment (ggabernet) [reply docs/images/nf-core-epitopeprediction_logo_dark.png:1]: ah wait I have an idea on how to solve it

[the remark this answers is not in the document]
Comment (ggabernet) [reply docs/images/nf-core-epitopeprediction_logo_dark.png:1]: nope it also did not work :(

[the remark this answers is not in the document]
Comment (ggabernet) [reply docs/images/nf-core-epitopeprediction_logo_dark.png]: I've just copied the logo light now and called it dark, so there is at least the logo

**Pull request merged**